### PR TITLE
Remove useless lines in commands and set some configuration.

### PR DESCRIPTION
**Pull request by jhryniuk**
@partikus CR please.

### CODE REVIEW

File: src/AppBundle/Entity/Employee.php [hunk 1]
```diff
@@ -82,13 +82,23 @@ class Employee
...
+     *     mappedBy="employee",
+     *     orphanRemoval=true
+     * )
      */
     private $skills;
 
     /**
      * @var Language
-     * @ORM\OneToMany(targetEntity="AppBundle\Entity\EmployeeLanguage", cascade={"ALL"}, mappedBy="employee", orphanRemoval=true)
+     * @ORM\OneToMany(
```
Comment (partikus) [inline src/AppBundle/Entity/Employee.php]: move that to another PR fix

File: src/AppBundle/Entity/Employee.php [hunk 2]
```diff
@@ -285,7 +295,7 @@ public function getProjects()
 
     public function addProject(Project $project)
     {
-        if ( ! $this->projects->contains($project)) {
+        if (! $this->projects->contains($project)) {
```
Comment (partikus) [inline src/AppBundle/Entity/Employee.php]: sometimes yo use `! $this` or `!$this`, please choose only one

File: src/AppBundle/Entity/Project.php [hunk 3]
```diff
@@ -78,7 +78,12 @@ class Project
 
     /**
      * @var ArrayCollection|ProjectSkill[]
-     * @ORM\OneToMany(targetEntity="AppBundle\Entity\ProjectSkill", cascade={"ALL"}, mappedBy="project", orphanRemoval=true)
+     * @ORM\OneToMany(
```
Comment (partikus) [inline src/AppBundle/Entity/Project.php]: idem, separate to another PR

**Comment by partikus** [comment]
@jhryniuk could you provide travis configuration to run code sniffer? please move phpunit to another PR

**Comment by jhryniuk** [comment]
Fexed.

**Comment by partikus** [comment]
@jhryniuk I meant sth like that: https://github.com/symfony/symfony-demo/blob/master/.travis.yml 

feel free to ask if you need help

**Comment by jhryniuk** [comment]
I create that one, and I believe that it is good.
Any advice's nice to see.

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+cache:
+  directories:
+    - $HOME/.composer/cache/files
+
+matrix:
+  fast_finish: true
+  include:
+    - php: 5.5
+    - php: 5.6
+    - php: 7.0
```
Comment (partikus) [inline .travis.yml]: @jhryniuk please remove `5.*` version, keep only `7.0` and add `7.1` with allowed failures

Comment (janhryniukpgs) [inline .travis.yml]: Ok. I'll fix it this evening

File: .travis.yml [hunk 5]
```diff
@@ -18,4 +18,4 @@ install:
   - composer install
 
 script:
-  - ./vendor/bin/phpcs --standard=PSR2 src
+  - ./vendor/bin/phpcs -n --standard=PSR2 src
```
Comment (partikus) [inline .travis.yml]: @jhryniuk +1

**Comment by partikus** [comment]
@jhryniuk so please switch travis to `ant` or `phing` but maybe in the next PR

**Comment by jhryniuk** [comment]
yes, I agree :+1:

**Comment by jhryniuk** [comment]
Now travis use ant as test manager. IMO that is good solution. What is your opinion about that?
